utils/utils.go

```diff
@@ -24,12 +25,29 @@ func Min(a, b int) int {
 	return b
 }
 
+func openInBrowser(url string) error {
```

Comment:
I actually didn't notice this function was just for linux, so please rename.
Also you forgot to change the call online 50
```suggestion
func openLinuxBrowser(url string) error {
```

Reply:
Oh, sorry about that